gp-translation-helpers.php

```diff
@@ -40,3 +40,5 @@
 add_action( 'gp_init', array( 'WPorg_GlotPress_Notifications', 'init' ) );    // todo: include this class in a different plugin.
 add_action( 'gp_init', array( 'WPorg_GlotPress_Customizations', 'init' ) );    // todo: include this class in a different plugin.
 add_filter( 'gp_enable_changesrequested_status', '__return_true' ); // todo: remove this filter when this plugin will be merged in the GlotPress core.
+add_action( 'gp_init', array( 'WPorg_GlotPress_Customizations', 'init' ) );
+add_filter( 'jetpack_mentions_should_load_ui', '__return_true' );
```

Comment:
This line should probably rather go into `WPorg_GlotPress_Notifications::init()`